mobile/asa-go/src/components/profile/CriticalHours.tsx

```diff
@@ -0,0 +1,25 @@
+import { Typography } from '@mui/material'
+import React from 'react'
+import { isNil } from 'lodash'
+import { formatCriticalHoursTimeText } from '@/utils/criticalHoursStartEndTime'
+
+interface CriticalHoursProps {
+  start?: number
+  end?: number
+}
+
+const CriticalHours = ({ start, end }: CriticalHoursProps) => {
+  let formattedCriticalHours = '-'
+
+  if (!isNil(start) && !isNil(end)) {
+    const [formattedStartTime, formattedEndTime] = formatCriticalHoursTimeText(start, end)
+    formattedCriticalHours = `${formattedStartTime} - ${formattedEndTime}`
+  }
+  return (
+    <Typography sx={{ fontSize: '0.75rem' }} data-testid="critical-hours">
```

Comment:
If we add `display: "flex", height: "100%", alignItems: "center"` to the Typography sx prop the text should be centered vertically.

Reply:
https://github.com/bcgov/wps/pull/4708/commits/f465787707ea56ae6c04e22b9d6cafc360fa1668